Error in printf (error: format not a string literal and no format arguments)

Trying to build the deb package, got this:

```
g++  -I../../core -I../../core/deps -I../../core/khronos -I../linux-deps/include -g -O3  -D RELEASE -c -D TARGET_LINUX_x86 -D USES_HOMEDIR -D HOST_NO_REC  -frename-registers -fno-strict-aliasing -fsingle-precision-constant  -ffast-math -ftree-vectorize  -g -O2 -fstack-protector --param=ssp-buffer-size=4 -Wformat -Werror=format-security  -fno-exceptions -fno-rtti -D SUPPORT_X11  -fomit-frame-pointer ../../core/hw/sh4/interpr/sh4_opcodes.cpp -o obj/hw/sh4/interpr/sh4_opcodes.build_obj
../../core/hw/sh4/interpr/sh4_opcodes.cpp: In function ‘void cpu_iNimp(u32, const char*)’:
../../core/hw/sh4/interpr/sh4_opcodes.cpp:65:13: error: format not a string literal and no format arguments [-Werror=format-security]
  printf(info);
             ^
../../core/hw/sh4/interpr/sh4_opcodes.cpp: In function ‘void cpu_iWarn(u32, const char*)’:
../../core/hw/sh4/interpr/sh4_opcodes.cpp:73:13: error: format not a string literal and no format arguments [-Werror=format-security]
  printf(info);
             ^
cc1plus: some warnings being treated as errors
make[1]: *** [obj/hw/sh4/interpr/sh4_opcodes.build_obj] Error 1
make[1]: Leaving directory `/home/sergio/Projects/reicast/reicast-emulator/shell/lin86'
```

Workaround: export -Wno-format-security to CFLAGS

--- a/core/hw/sh4/interpr/sh4_opcodes.cpp
+++ b/core/hw/sh4/interpr/sh4_opcodes.cpp
@@ -70,7 +70,7 @@ void cpu_iNimp(u32 op, const char* info)
 void cpu_iWarn(u32 op, const char* info)
 {
 	printf("Check opcode : %X : ", op);
-	printf(info);
+	printf("%s", info);
 	printf(" @ %X\n", curr_pc);
 }
 

